scripts/3.TCGA-MLexample_Pathway.py

```diff
@@ -313,11 +319,21 @@ def get_threshold_metrics(y_true, y_pred, tissue='all'):
 metrics_test = get_threshold_metrics(y_test, y_pred_test)
 
 
-# In[33]:
+# In[34]:
+
+# Rerun "cross validation" for the best hyperparameter set to define
+# cross-validation disease-specific performance. Each sample prediction is
+# based on the fold that the sample was in the testing partition
+y_cv = cross_val_predict(cv_pipeline.best_estimator_, X=X_train, y=y_train,
```

Comment:
Didn't know this was standard practice, as you are grouping predictions that were made from different models. However, given that sklearn has a `cross_val_predict` function, I'll accept its validity. The alternative would be to draw an ROC curve for each CV-fold, but this is much prettier and concise. It appears to work here.